src/stores/GithubStore.js

```diff
@@ -46,15 +45,41 @@ export default class GithubStore {
       throw err;
     }
 
-    if (response.content) {
-      return Buffer.from(response.content, 'base64').toString().trim();
+    let { content } = response;
+
+    if (content) {
+      content = (decode ? Buffer.from(content, 'base64').toString() : content).trim();
+
+      if (render) {
+        return this.renderer.render(content, {
```

Comment:
I think we can keep this outside of the store, and get rid of the `selfRenders` option.  We always know the `contentPath` outside of the store, and the store can still expose `getContentFromPath`.